spec/services/get_products_service_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe GetProductsService do
+  subject(:service) { described_class.new(category_id) }
+
+  let(:category_id) { SecureRandom.uuid }
+  let(:all_category_id) { SecureRandom.uuid }
```

Comment:
why don't u name it like 'trà id', or 'trà sữa id', which will make more sense than first and second parent cateogry